src/util/TickUtils.ts

```diff
@@ -0,0 +1,78 @@
+import _ from 'lodash';
+
+import { TickFormatter } from '../cartesian/CartesianAxis';
+import { getAngledRectangleWidth } from './CartesianUtils';
+import { getStringSize } from './DOMUtils';
+import { getEveryNthWithCondition } from './getEveryNthWithCondition';
+import { Size, CartesianViewBox, CartesianTickItem } from './types';
+
+export function getAngledTickWidth(contentSize: Size, unitSize: Size, angle: number) {
+  const size = { width: contentSize.width + unitSize.width, height: contentSize.height + unitSize.height };
```

Comment:
sanity check: ideally these won't be NaN or undefined, but calling code needs to ensure that

Reply:
I only moved this function from one file to another. I would prefer to not extend this. 
I do agree with the sentiment though.